frontend/src/metabase/visualizations/echarts/tooltip/index.tsx

```diff
@@ -70,14 +71,18 @@ export const getTooltipBaseOption = (
       let container = document.querySelector(
         ".echarts-tooltip-container",
       ) as HTMLDivElement;
+
       if (!container) {
         container = document.createElement("div");
         container.classList.add("echarts-tooltip-container");
         container.style.setProperty("overflow", "hidden");
-        container.style.setProperty("position", "absolute");
+        container.style.setProperty("position", "fixed");
         container.style.setProperty("inset", "0");
         container.style.setProperty("pointer-events", "none");
 
+        const tooltipContainerZIndex = `var(--mb-overlay-z-index, ${OVERLAY_Z_INDEX})`;
```

Comment:
Use the `--mb-overlay-z-index` z-index value if overrides are applied via the SDK theme configuration, otherwise use the default `OVERLAY_Z_INDEX` value as a fallback. We need to add the fallback here because we did not set `--mb-overlay-z-index` by default.

Reply:
@WiNloSt Definitely, this was on my mind too. Now that I check the code more carefully, we actually might be able to simply remove the `OVERLAY_Z_INDEX` fallback here as Raffi already added it as `popover.zIndex`'s default:

https://github.com/metabase/metabase/blob/0db07a59977069db6d1ea3dfadff60e819070fc4/frontend/src/metabase/embedding-sdk/theme/default-component-theme.ts#L92-L95

And he already mapped it here:

https://github.com/metabase/metabase/blob/0db07a59977069db6d1ea3dfadff60e819070fc4/frontend/src/metabase/styled-components/theme/css-variables.ts#L25

Let me try to remove it and see if it works.